bindings/pydeck/pydeck/frontend_semver.py

```diff
@@ -0,0 +1,2 @@
+"""Current associated version of NPM modules deck.gl and @deck.gl/jupyter-widget"""
```

Comment:
todo: read from lerna.json instead

Reply:
also check that the correct tag matches the current branch (beta/alpha from master only)